universe/Pathfinder.h

```diff
@@ -0,0 +1,136 @@
+#ifndef _Pathfinder_h_
+#define _Pathfinder_h_
+
+#include "UniverseObject.h"
+
+#include <boost/unordered_map.hpp>
+
+#include <vector>
+#include <map>
+#include <set>
+
+#ifdef FREEORION_WIN32
```

Comment:
seems to build OK for me without this `#undef` 
https://github.com/freeorion/freeorion/pull/1249/files#diff-1567159526cce829c7e2802017835646block